### PR TITLE
Oct 22 firedrill changes

### PR DESCRIPTION
**Pull request by tsengj10**
Some changes prompted by the Oct 2022 firedrill:
- added more informative error messages to app.py, to make it easier to catch configuration errors and json parsing problems
- added fd2210.csv, a configuration which illustrates copying the _id field to burst_id, and then using it in the output filenames. This should eliminate the problem of snewpdag overwriting files for a quick succession of SNEWS alerts.

To be done:  null in the json input.

### CODE REVIEW

**Comment by tsengj10** [comment]
Done:  input data will be interpreted using json.loads, which can parse fairly standard JSON, including the null keyword.

Configuration (such as in .py files and the argument strings of .csv) still uses python literal evaluation, as it's less fiddly and more python-esque.

File: snewpdag/data/fd2210.csv [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+"prob","Chi2Prob","Diff","'in_field':'map', 'out_field':'probmap'"
+"prob-out","Pass","prob","'line':1, 'dump':1"
+"probmap","renderers.Mollview","prob","'in_field':'probmap', 'title':'DiffPointing probability', 'units':'probability', 'coord':['C'], 'filename':'output/fd2210-{0}-{2}.png'"
+"fits","renderers.FitsSkymap","prob","'in_field':'probmap', 'filename':'output/fd2210-{0}-{2}.fits'"
+,,,
+"conf","Chi2CL","Diff","'in_field':'map', 'out_field':'clmap'"
+"conf-out","Pass","conf","'line':1, 'dump':1"
+"confmap","renderers.Mollview","conf","'in_field':'clmap', 'title':'DiffPointing CL', 'units':'CL', 'range':(0,1), 'coord':['C'], 'filename':'output/fd2210-{0}-{2}.png'"
+"conffits","renderers.FitsSkymap","conf","'in_field':'clmap', 'filename':'output/fd2210-{0}-{2}.fits'"
+,,,
```
Comment (mcolomermolla) [inline snewpdag/data/fd2210.csv:23]: Minor suggestion:

As "_id" already contains the date, I would avoid having "fd2210" in the output filename, maybe just "fd-out.."?

Comment (tsengj10) [reply snewpdag/data/fd2210.csv:23]: I kept the fd2210 there so it was obvious which configuration script was used.

**Comment by mcolomermolla** [comment]
Issue solved with last commit. I think all points raised from last firedrill are implemented in this PR and we are ready to use this changes in next one. We can plan some self tests before that for cross-check of the pointing result.

**Pull request merged**